bip-0001.mediawiki

```diff
@@ -25,6 +25,8 @@ There are three kinds of BIP:
 
 The BIP editors assign BIP numbers and change their status. Please send all BIP-related email to the BIP editor, which is listed under [[#BIP_Editors|BIP Editors]] below. Also see [[#BIP_Editor_Responsibilities__Workflow|BIP Editor Responsibilities & Workflow]].
 
+Authors MUST NOT self assign numbers, but may use aliases such as "bip-exampleproposal".
```

Comment:
I have a feeling if someone is reading this BIP they are not the type of person that self-assigns.

The readme is the best place for this notice and it's already there.